server/index.spec.js

```diff
@@ -727,6 +727,19 @@ describe('Server integration tests', function() {
         });
     });
 
+    it('Should include ETag in Access-Control-Expose-Headers in response', async function() {
+      await request(app)
+        .get('/me/shops')
+        .set('Authorization', `token ${dopplerApiKey}`)
+        .set('User-Agent', 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10.14; rv:68.0) Gecko/20100101 Firefox/68.0')
+        .set('Accept', '*/*')
+        .set('Referer', 'https://app.fromdoppler.com/')
+        .set('Origin', 'https://app.fromdoppler.com')
+        .expect(function(res) {
+          expect(res.get('Access-Control-Expose-Headers')).to.be.eql('ETag');

```

Comment:
Great! Could you also verify here that the actual header property `Etag` is currently there?

Reply:
Mmm, it is out of the scope of the change, I do not know it ETag is there or not, it is independent of CORS configuration.

Although, I can add some tests for ETag.